src/sider/1-forside/seksjoner/kategorier/Skjemaveilederlenke.tsx

```diff
@@ -1,22 +1,35 @@
-import { getSkjemaveilederUrlBedrift, getSkjemaveilederUrlPerson } from "../../../../config";
+import {
+  getSkjemaveilederUrlBedrift,
+  getSkjemaveilederUrlPerson
+} from "../../../../config";
 import React from "react";
 import { InjectedIntlProps, injectIntl, FormattedMessage } from "react-intl";
 
 interface Props {
-    type: string;
+  type: string;
 }
 
 const Skjemaveilederlenke = (props: Props & InjectedIntlProps) => (
-    <div className="skjemaveilederlenke">
-        {props.type === "bedrift" ?
-            <a className="lenke" href={getSkjemaveilederUrlBedrift()}>
-                <FormattedMessage id="skjemaveilederlenke.bedrift" />
-            </a> :
-            <a className="lenke" href={getSkjemaveilederUrlPerson(props.intl.locale)}>
-                <FormattedMessage id="skjemaveilederlenke.person" />
-            </a>
-        }
-    </div>
+  <section className="header__wrapper seksjon oversikt">
+    <section className="filler" />
+    <section className="innhold__container">
```

Comment:
Tre wrappede sections er vel ikke akkurat nydelig, men det var ei heller utseende på den, og den forsvinner om to uker, så 👍

Reply:
Helt enig, men det ble en quickfix utseendemessig frem til den forsvinner om to uker